src/main.mjs

```diff
@@ -103,9 +103,9 @@ export default class ImgixClient {
   }
 
   buildSrcSet(path, params = {}, options = {}) {
-    const { w, h, ar } = params;
+    const { w, h } = params;
 
-    if (w || (h && ar)) {
+    if (w || h) {
```

Comment:
Little nit that a truthy check should never be used with numbers. Instead, this should be:

```js
if (w != null || h != null) {
```